pokemon-react-app/src/styles/Ball.style.js

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components'
+
+export const Ball = styled.div`
```

Comment:
General obs: The styles for each component should be in a file near the component itself, like `PokemonDetail.style.js`